x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/evaluate_panel.tsx

```diff
@@ -177,27 +301,32 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus }) => {
               )}
             </span>
           </EuiTitle>
+          {trainingDocsCount !== null && (
+            <EuiText size="xs" color="subdued">
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.regressionExploration.trainingDocsCount"
+                defaultMessage="{docsCount} docs evaluated"
```

Comment:
As above, this should include syntax for one / plural cases, for when there is 1 doc.